.travis.yml

```diff
@@ -2,37 +2,39 @@ language: cpp
 sudo: required
 dist: trusty
 
-compiler:
-  - gcc
+compiler:  # A trick to construct a build matrix
+  - printf "%s" "3.6" >> ~/PYTHON_VERSION ; gcc
+
+os:
+  - linux
+  - osx
+
+env:
+  matrix:
+    - TASK=regular
+    - TASK=mpi
+    - TASK=pylint
+    - TASK=if-else
+    - TASK=sdist
+    - TASK=bdist
+    - TASK=gpu METHOD=source
+    - TASK=gpu METHOD=pip
+    - TASK=regular PYTHON_VERSION=2.7
+    - TASK=regular PYTHON_VERSION=3.4
+    - TASK=regular PYTHON_VERSION=3.5
```

Comment:
why do you use 3.5, not 3.6?

Reply:
@henry0312 3.6 is default one. Why we need the same task one more time?

Reply:
I'm sorry I misunderstood.